verilog/analysis/verilog_analyzer.h

```diff
@@ -23,6 +23,7 @@
 #include "absl/status/status.h"
 #include "absl/strings/string_view.h"
 #include "common/analysis/file_analyzer.h"
+#include "common/strings/mem-block.h"
```

Comment:
Not in this PR, but perhaps rename file to `mem_block.h` for consistency with the rest of the project?

Reply:
(putting on TODO list)